indico/migrations/versions/20201209_2010_e4fb983dc64c_modification_mode_enum.py

```diff
@@ -0,0 +1,29 @@
+"""modification_mode enum
```

Comment:
That's not really a useful revision title. Let's change this to something like `Add 'while pending' regform modification mode` and rename the file to `20201209_2010_e4fb983dc64c_add_while_pending_regform_modification_mode.py`